### PR TITLE
https://issues.openmrs.org/browse/TRUNK-5821

### PR DESCRIPTION
**Pull request by mogoodrich**
(Move to 2.3.x line)

## Issue I worked on

see https://issues.openmrs.org/browse/TRUNK-

## Checklist: I completed these to help reviewers :)

- [ ] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [ ] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [ ] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [ ] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [ ] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

File: api/src/main/resources/org/openmrs/liquibase/updates/liquibase-update-to-latest-2.4.x.xml [hunk 1]
```diff
@@ -299,34 +299,5 @@
...
-		<preConditions onFail="MARK_RAN" onFailMessage="Table orders does not exist or the index orders_order_number already exists.">
-			<tableExists tableName="orders"/>
-			<and>
-				<not>
-					<indexExists indexName="orders_order_number" />
-				</not>
-			</and>
-		</preConditions>
-		<comment>Adding index to order_number column in Orders table</comment>
-		<createIndex  indexName="orders_order_number" tableName="orders">
```
Comment (dkayiwa) [inline api/src/main/resources/org/openmrs/liquibase/updates/liquibase-update-to-latest-2.4.x.xml:313]: Only one liquibase-update-to-latest-version.xml file is run.
Therefore, removing the changesets from this file means that they will never be run on the current master branch.

Comment (mogoodrich) [reply api/src/main/resources/org/openmrs/liquibase/updates/liquibase-update-to-latest-2.4.x.xml:313]: Thanks @dkayiwa !  So the correct thing to do would be to have these changesets in both update-to-latest-2.4.x and update-to-latest-2.3.x?

Comment (dkayiwa) [reply api/src/main/resources/org/openmrs/liquibase/updates/liquibase-update-to-latest-2.4.x.xml:313]: Tru dat! 👍

Comment (mogoodrich) [reply api/src/main/resources/org/openmrs/liquibase/updates/liquibase-update-to-latest-2.4.x.xml]: Thanks!  I updated this, merging now...

**Pull request merged**